src/GUI/GUI.java

```diff
@@ -236,8 +234,16 @@ private static void addDropdownWithFilters() {
             @Override
             public void itemStateChanged(ItemEvent e) {
```

Comment:
den code block und den identischen code block da drunter mal genauer beschreiben was da überhaupt passiert sonst checkts keiner in Minuten